packages/flutter/test/widgets/input_test.dart

```diff
@@ -498,14 +476,14 @@ void main() {
     Size threeLineInputSize = inputBox.size;
 
     // An extra line won't increase the size because we max at 3.
-    enterText(kFourLines);
+    await tester.enterText(find.byType(EditableText).first, kFourLines);
```

Comment:
I just lifted that expression from the existing code; happy to replace must (all?) occurrences with find.byType(EditableText).